mopidy/local/commands.py

```diff
@@ -61,7 +61,10 @@ def __init__(self):
         super(ScanCommand, self).__init__()
         self.add_argument('--limit',
                           action='store', type=int, dest='limit', default=None,
-                          help='Maxmimum number of tracks to scan')
+                          help='Maximum number of tracks to scan')
+        self.add_argument('--force',
+                          action='store_true',dest='force',default=False,
```

Comment:
Missing spaces after the commas here. Will brake flake8 linting when Travis CI gets up to speed again.